### PR TITLE
Add flakes support and rename secondary scripts

### PR DESCRIPTION
**Pull request by SenchoPens**
Hi, I added flakes so anyone can now install the scripts as a package on a NixOS.
Also I thought renaming secondary scripts so they begin with "nix-autobahn" won't harm, just so one can put them in PATH and not worry about name collisions.

### CODE REVIEW

**Comment by SenchoPens** [comment]
Oh, I think it will be good to put dependencies in the package, I'll do this later.

File: flake.nix [hunk 1]
```diff
@@ -0,0 +1,27 @@
+{
+  description = "My haskell application";
```
Comment (Lassulus) [inline flake.nix]: not sure if the description is correct?

Comment (SenchoPens) [reply flake.nix]: Oops. Changed it to "Shell script collection to download ELF binaries and use them right away!".

**Comment by Lassulus** [comment]
ah forgot about that, looks good I guess

**Pull request merged**